.env

```
@@ -48,12 +48,14 @@ WS_SERVER_OUTPUT_VERBOSITY=32
 WS_SERVER_PAYLOAD_DUMP=0
 
 # msp auth server
-AUTH_SERVER_SCHEME=https
-AUTH_SERVER_HOST=auth.mspchallenge.info
-AUTH_SERVER_PORT=443
+AUTH_SERVER_SCHEME=http
+AUTH_SERVER_HOST=localhost
+AUTH_SERVER_PORT=8000
```

I suppose we need to change this to the live one..

For dev we can set it in .env.local

```suggestion
AUTH_SERVER_SCHEME=https
AUTH_SERVER_HOST=auth2.mspchallenge.info
AUTH_SERVER_PORT=443
```